<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>R: Population Genetics</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link rel="stylesheet" type="text/css" href="../../../doc/html/R.css">
</head><body>
<h1> Population Genetics
<img class="toplogo" src="../../../doc/html/logo.jpg" alt="[R logo]"></h1>
</h1>
<hr>
<div align="center">
<a href="../../../doc/html/packages.html"><img src="../../../doc/html/left.jpg" alt="[Up]" width="30" height="30" border="0"></a>
<a href="../../../doc/html/index.html"><img src="../../../doc/html/up.jpg" alt="[Top]" width="30" height="30" border="0"></a>
</div><h2>Documentation for package &lsquo;genetics&rsquo; version 1.3.6</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">Overview of user guides and package vignettes</a>; browse <a href="../doc">directory</a>.</li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<p align="center">
<a href="#"></a>
<a href="#A">A</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#G">G</a>
<a href="#H">H</a>
<a href="#I">I</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#O">O</a>
<a href="#P">P</a>
<a href="#S">S</a>
<a href="#W">W</a>
<a href="#misc">misc</a>
</p>


<h2><a name="">--  --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="genotype.html">%in%</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">%in%.default</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">%in%.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">%in%.haplotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">==.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">==.haplotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
</table>

<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="homozygote.html">allele</a></td>
<td>Extract Features of Genotype objects</td></tr>
<tr><td width="25%"><a href="homozygote.html">allele.count</a></td>
<td>Extract Features of Genotype objects</td></tr>
<tr><td width="25%"><a href="undocumented.html">allele.count.2.genotype</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="genotype.html">allele.count.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="homozygote.html">allele.names</a></td>
<td>Extract Features of Genotype objects</td></tr>
<tr><td width="25%"><a href="locus.html">as.character.gene</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">as.character.locus</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">as.character.marker</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="undocumented.html">as.factor</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="undocumented.html">as.factor.allele.genotype</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="undocumented.html">as.factor.default</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="undocumented.html">as.factor.genotype</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="genotype.html">as.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">as.genotype.allele.count</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">as.genotype.character</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">as.genotype.default</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">as.genotype.factor</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">as.genotype.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">as.genotype.haplotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">as.genotype.table</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">as.haplotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="homozygote.html">carrier</a></td>
<td>Extract Features of Genotype objects</td></tr>
<tr><td width="25%"><a href="homozygote.html">carrier.genotype</a></td>
<td>Extract Features of Genotype objects</td></tr>
<tr><td width="25%"><a href="ci.balance.html">ci.balance</a></td>
<td>Experimental Function to Correct Confidence Intervals At or Near
  Boundaries of the Parameter Space by 'Sliding' the Interval on the
  Quantile Scale.</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="diseq.html">diseq</a></td>
<td>Estimate or Compute Confidence Interval for the Single-Marker Disequilibrium</td></tr>
<tr><td width="25%"><a href="diseq.html">diseq.ci</a></td>
<td>Estimate or Compute Confidence Interval for the Single-Marker Disequilibrium</td></tr>
<tr><td width="25%"><a href="diseq.html">diseq.genotype</a></td>
<td>Estimate or Compute Confidence Interval for the Single-Marker Disequilibrium</td></tr>
<tr><td width="25%"><a href="diseq.html">diseq.table</a></td>
<td>Estimate or Compute Confidence Interval for the Single-Marker Disequilibrium</td></tr>
</table>

<h2><a name="E">-- E --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="expectedGenotypes.html">expectedGenotypes</a></td>
<td>Construct expected genotypes/haplotypes according to known allele
  variants</td></tr>
<tr><td width="25%"><a href="expectedGenotypes.html">expectedHaplotypes</a></td>
<td>Construct expected genotypes/haplotypes according to known allele
  variants</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="locus.html">gene</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">gene&lt;-</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="undocumented.html">geno.as.array</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="genotype.html">genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="order.genotype.html">genotypeOrder</a></td>
<td>Order/sort genotype/haplotype object</td></tr>
<tr><td width="25%"><a href="order.genotype.html">genotypeOrder&lt;-</a></td>
<td>Order/sort genotype/haplotype object</td></tr>
<tr><td width="25%"><a href="locus.html">getgene</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">getlocus</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">getmarker</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="gregorius.html">gregorius</a></td>
<td>Probability of Observing All Alleles with a Given Frequency in a
  Sample of a Specified Size.</td></tr>
<tr><td width="25%"><a href="groupGenotype.html">groupGenotype</a></td>
<td>Group genotype values</td></tr>
</table>

<h2><a name="H">-- H --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="undocumented.html">hap</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="undocumented.html">hapambig</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="undocumented.html">hapenum</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="undocumented.html">hapfreq</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="genotype.html">haplotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="undocumented.html">hapmcmc</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="undocumented.html">hapshuffle</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="homozygote.html">heterozygote</a></td>
<td>Extract Features of Genotype objects</td></tr>
<tr><td width="25%"><a href="genotype.html">heterozygote.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="homozygote.html">homozygote</a></td>
<td>Extract Features of Genotype objects</td></tr>
<tr><td width="25%"><a href="genotype.html">homozygote.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="HWE.chisq.html">HWE.chisq</a></td>
<td>Perform Chi-Square Test for Hardy-Weinberg Equilibrium</td></tr>
<tr><td width="25%"><a href="HWE.chisq.html">HWE.chisq.genotype</a></td>
<td>Perform Chi-Square Test for Hardy-Weinberg Equilibrium</td></tr>
<tr><td width="25%"><a href="HWE.exact.html">HWE.exact</a></td>
<td>Exact Test of Hardy-Weinberg Equilibrium for 2-Allele Markers</td></tr>
<tr><td width="25%"><a href="HWE.test.html">HWE.test</a></td>
<td>Estimate Disequilibrium and Test for Hardy-Weinberg Equilibrium</td></tr>
<tr><td width="25%"><a href="HWE.test.html">HWE.test.data.frame</a></td>
<td>Estimate Disequilibrium and Test for Hardy-Weinberg Equilibrium</td></tr>
<tr><td width="25%"><a href="HWE.test.html">HWE.test.genotype</a></td>
<td>Estimate Disequilibrium and Test for Hardy-Weinberg Equilibrium</td></tr>
</table>

<h2><a name="I">-- I --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="locus.html">is.gene</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="genotype.html">is.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">is.haplotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="locus.html">is.locus</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">is.marker</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
</table>

<h2><a name="L">-- L --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="LD.html">LD</a></td>
<td>Pairwise linkage disequilibrium between genetic markers.</td></tr>
<tr><td width="25%"><a href="LD.html">LD.data.frame</a></td>
<td>Pairwise linkage disequilibrium between genetic markers.</td></tr>
<tr><td width="25%"><a href="LD.html">LD.genotype</a></td>
<td>Pairwise linkage disequilibrium between genetic markers.</td></tr>
<tr><td width="25%"><a href="print.LD.html">LDplot</a></td>
<td>Textual and graphical display of linkage disequilibrium (LD) objects</td></tr>
<tr><td width="25%"><a href="print.LD.html">LDtable</a></td>
<td>Textual and graphical display of linkage disequilibrium (LD) objects</td></tr>
<tr><td width="25%"><a href="locus.html">locus</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">locus&lt;-</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="makeGenotypes.html">makeGenotypes</a></td>
<td>Convert columns in a dataframe to genotypes or haplotypes</td></tr>
<tr><td width="25%"><a href="makeGenotypes.html">makeHaplotypes</a></td>
<td>Convert columns in a dataframe to genotypes or haplotypes</td></tr>
<tr><td width="25%"><a href="locus.html">marker</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">marker&lt;-</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="undocumented.html">mknum</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="undocumented.html">mourant</a></td>
<td>Undocumented functions</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="genotype.html">nallele</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
</table>

<h2><a name="O">-- O --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="order.genotype.html">order</a></td>
<td>Order/sort genotype/haplotype object</td></tr>
<tr><td width="25%"><a href="order.genotype.html">order.genotype</a></td>
<td>Order/sort genotype/haplotype object</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="plot.genotype.html">plot.genotype</a></td>
<td>Plot genotype object</td></tr>
<tr><td width="25%"><a href="print.LD.html">plot.LD.data.frame</a></td>
<td>Textual and graphical display of linkage disequilibrium (LD) objects</td></tr>
<tr><td width="25%"><a href="depreciated.html">power.casectrl</a></td>
<td>Depreciated functions</td></tr>
<tr><td width="25%"><a href="genotype.html">print.allele.count</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">print.allele.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="diseq.html">print.diseq</a></td>
<td>Estimate or Compute Confidence Interval for the Single-Marker Disequilibrium</td></tr>
<tr><td width="25%"><a href="locus.html">print.gene</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="genotype.html">print.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="HWE.test.html">print.HWE.test</a></td>
<td>Estimate Disequilibrium and Test for Hardy-Weinberg Equilibrium</td></tr>
<tr><td width="25%"><a href="print.LD.html">print.LD</a></td>
<td>Textual and graphical display of linkage disequilibrium (LD) objects</td></tr>
<tr><td width="25%"><a href="print.LD.html">print.LD.data.frame</a></td>
<td>Textual and graphical display of linkage disequilibrium (LD) objects</td></tr>
<tr><td width="25%"><a href="locus.html">print.locus</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="locus.html">print.marker</a></td>
<td>Create and Manipulate Locus, Gene, and Marker Objects</td></tr>
<tr><td width="25%"><a href="summary.genotype.html">print.summary.genotype</a></td>
<td>Allele and Genotype Frequency from a Genotype or
  Haplotype Object</td></tr>
<tr><td width="25%"><a href="print.LD.html">print.summary.LD.data.frame</a></td>
<td>Textual and graphical display of linkage disequilibrium (LD) objects</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="undocumented.html">shortsummary.genotype</a></td>
<td>Undocumented functions</td></tr>
<tr><td width="25%"><a href="order.genotype.html">sort.genotype</a></td>
<td>Order/sort genotype/haplotype object</td></tr>
<tr><td width="25%"><a href="summary.genotype.html">summary.genotype</a></td>
<td>Allele and Genotype Frequency from a Genotype or
  Haplotype Object</td></tr>
<tr><td width="25%"><a href="print.LD.html">summary.LD.data.frame</a></td>
<td>Textual and graphical display of linkage disequilibrium (LD) objects</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="write.pop.file.html">write.marker.file</a></td>
<td>Create genetics data files</td></tr>
<tr><td width="25%"><a href="write.pop.file.html">write.pedigree.file</a></td>
<td>Create genetics data files</td></tr>
<tr><td width="25%"><a href="write.pop.file.html">write.pop.file</a></td>
<td>Create genetics data files</td></tr>
</table>

<h2><a name="misc">-- misc --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="genotype.html">[.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">[.haplotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">[&lt;-.genotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
<tr><td width="25%"><a href="genotype.html">[&lt;-.haplotype</a></td>
<td>Genotype or Haplotype Objects.</td></tr>
</table>
</body></html>
